docs/flink/flink-getting-started.md

```diff
@@ -300,9 +301,9 @@ CREATE TABLE `hive_catalog`.`default`.`sample` (
 );
 ```
 
-Table create commands support the most commonly used [flink create clauses](https://ci.apache.org/projects/flink/flink-docs-release-1.11/dev/table/sql/create.html#create-table) now, including: 
+Table create commands support the most commonly used [flink create clauses](https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/dev/table/sql/create/) now, including: 
```

Comment:
```suggestion
Table create commands support the most commonly used [flink create clauses](https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/dev/table/sql/create/), including: 
```
(nit)

Reply:
@samredai Thank you for your reply. I have changed the all the advise.